WebRoot/skins/_base/base4/skin.css

```diff
@@ -715,6 +715,15 @@ SELECT                      {   @NativeSelect@  }
     @Text-unread@
 }
 
+.Row .ZmConvListFrom,
```

Comment:
can we apply this style directly to ZmConvListFrom? it looks like in every case we are applying this

Reply:
If you see few of the lines above (https://github.com/Zimbra/zm-web-client/blob/56d075ca1abe1b7a1db915538303a121a8bfceaf/WebRoot/skins/_base/base4/skin.css#L689), you can see that selector is more specific and not just `ZmConvListFrom` and so we need to follow same way to override it.

Thanks

Reply:
Updated: https://github.com/Zimbra/zm-web-client/pull/206/files